content/The_Cartridge_Header.md

```diff
@@ -187,6 +187,11 @@ Specifies the ROM Size of the cartridge. Typically calculated as "N such that 32
 |`$06`|   2 MByte |128 banks|
 |`$07`|   4 MByte |256 banks|
 |`$08`|   8 MByte |512 banks|
+
+The following rom types are listed on older pandocs but the source is unknown, and no carts using them are known. You probably don't need to support these.
```

Comment:
Aren't they used by MMM01?

Reply:
The only known cart with that mapper is Momotarou Collection 2 - which is a 1MB ROM. Not 1.1, 1.2, or 1.5.

Reply:
Regarding other odd MBCs:
MBC6 is used in one cart, and has 1MB of rom and 1MB flash - not 1.1, 1.2, or 1.5
MBC7 - a handful of related carts, all also 1 MB